gitlab/utils.py

```diff
@@ -90,3 +93,30 @@ def __new__(  # type: ignore
 
 def remove_none_from_dict(data: Dict[str, Any]) -> Dict[str, Any]:
     return {k: v for k, v in data.items() if v is not None}
+
+
+def warn(
+    *,
+    message: str,
+    category: Optional[Type] = None,
+    source: Optional[Any] = None,
+) -> None:
```

Comment:
IMO we could have the signature the same as what's being wrapped, e.g. `message` is a positional. I think with one positional argument the intention is quite clear and a good middle ground in terms of explicitness/verbosity, as is done in stdlib in https://docs.python.org/3/library/warnings.html#warnings.warn.

Reply:
Changed to `message` being a positional argument.